spec/factories/user.rb

```diff
@@ -1,8 +1,17 @@
-# FactoryGirl.define do
-#   factory :user do
-#     name "John Doe"
-#     username  "JDoe"
-#     email "jdoe@example.com"
-#     password "password"
-#
-#   end
+FactoryGirl.define do
+  factory :default_user, class: User do
+    name      "John Doe"
+    username  "JDoe"
+    email     "johndoe@example.com"
+    password  "password"
+    role      0
+  end
+
+  factory :admin_user, class: User do
+    name      "Jane Doe"
+    username  "JaneDoe"
+    email     "janedoe@example.com"
+    password  "password"
+    role      1
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.